canary-release/action.yml

```diff
@@ -38,7 +38,7 @@ runs:
     - name: Set output variables
       id: vars
       shell: bash -l {0}
-      run: echo ::set-output name=run-url::"https://github.com/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID"
+      run: echo "name=run-url::https://github.com/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID" >> $GITHUB_OUTPUT
```

Comment:
```suggestion
      run: echo "RUN_URL=https://github.com/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID" >> $GITHUB_OUTPUT
```

Reply:
Wow, it is totally unclear in the github actions docs that the old syntax ?? uppercases the variable name and converts - to _? Really?